api/serializers.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth.models import User, Group
+from rest_framework import serializers
+from .models import Employee, Income, Outcome, Event
+
+
+class UserSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = User
+        fields = ('url', 'username', 'email', 'groups')
+
+
+class GroupSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = Group
+        fields = ('url', 'name')
+
+
+class EmployeeSerializer(serializers.HyperlinkedModelSerializer):
+
+    def validate(self, data):
```

Comment:
Instead of validating by hand, you should specify types of fields: https://www.django-rest-framework.org/api-guide/fields/#serializer-fields

I.e.

```python
class EmployeeSerializer(serializers.HyperlinkedModelSerializer):
	hired = serializer.BooleanField(default=True)

	(...)
```

Then the serializer will validate it for you. Please, read the documentation I linked above and use valid fields declarations.

Reply:
done!